pkg/cmd/create/install.go

```diff
@@ -1169,28 +1165,9 @@ func (options *InstallOptions) configureHelmRepo() error {
 
 func (options *InstallOptions) selectJenkinsInstallation() error {
 	if !options.BatchMode {
-		//install type has not been configured
-		if !options.Flags.Prow && !options.Flags.StaticJenkins {
-			jenkinsInstallOptions := []string{
-				ServerlessJenkins,
-				StaticMasterJenkins,
-			}
-			jenkinsInstallOption, err := util.PickNameWithDefault(jenkinsInstallOptions, "Select Jenkins installation type:", ServerlessJenkins, "", options.GetIOFileHandles())
-			if err != nil {
-				return errors.Wrap(err, "picking Jenkins installation type")
-			}
-			if jenkinsInstallOption == ServerlessJenkins {
-				options.Flags.Prow = true
-				options.Flags.Tekton = true
-			}
-		} else {
-			//determine which install type is configured
-			jenkinsInstallOption := ServerlessJenkins
-			if options.Flags.StaticJenkins {
-				jenkinsInstallOption = StaticMasterJenkins
-			}
-			log.Logger().Infof(util.QuestionAnswer("Configured Jenkins installation type", jenkinsInstallOption))
-		}
+		//determine which install type is configured
+		jenkinsInstallOption := ServerlessJenkins
+		log.Logger().Infof(util.QuestionAnswer("Configured Jenkins installation type", jenkinsInstallOption))
```

Comment:
This could be a bit misleading as well, since it somehow implies that there are other installation types, but that is not the case, right?

Reply:
Yeah, that could be better worded, but I think it's fine.